package.json

```
@@ -57,9 +58,9 @@
   "oclif": {
     "commands": "./src/commands",
     "bin": "aio",
```

You need to mark "aem:rde:inspect" as an experimental feature.

```suggestion
    "bin": "aio",
    "experimental-features": ["aem:rde:inspect"], 
```

---

@jsedding should i already enable it? because like that it is not enabled for me.

---

No, it should not yet be enabled by default.